nistats/reporting/glm_reporter.py

```diff
@@ -446,6 +452,8 @@ def _make_report_components(statistical_maps, contrasts_plots, threshold, alpha,
                                          min_distance=min_distance,
                                          )
         )
+        contrast_name = contrast_name.replace('<', '&lt')
+        contrast_name = contrast_name.replace('>', '&gt')
```

Comment:
these two were just examples. you shouldn't try to do this manually but use `html.escape` from the standard library

Reply:
Ah okay! :-)